src/vs/platform/menubar/electron-main/menubar.ts

```diff
@@ -293,15 +305,6 @@ export class Menubar {
 			menubar.append(macWindowMenuItem);
 		}
 
-		// Preferences
```

Comment:
@sbatten is that change related to this PR?

Reply:
yea this was leftover from before submenu support was added to the custom menubar and we made it a top-level item.